src/code/speed_meter.c

```diff
@@ -237,10 +248,9 @@ void SpeedMeter_DrawAllocEntries(SpeedMeter* meter, GraphicsContext* gfxCtx, Gam
         y++;
     }
 
-    thga = (TwoHeadGfxArena*)&state->tha;
-    //! @bug THA_GetRemaining call should be THGA_GetRemaining like the others below, harmless as-is
```

Comment:
Why is the bug comment gone?

Reply:
he changed the temp to be a tha instead of a thga, so its the right function call

Reply:
I was very confused as well, I think we missed the tha to thga cast back when you (tharo) overhauled th(g)a docs